assets/js/components/dashboard-sharing/DashboardSharingSettingsButton.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * DashboardSharingSettingsButton component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Dialog, { DialogContent, DialogFooter } from '@material/react-dialog';
```

Comment:
Thanks, @asvinb. All looks good to me. One last query, could you, please, move this material library import to the `material-components` folder for consistency with other material libraries usage: https://github.com/google/site-kit-wp/blob/1d620286114b079d452babb18bb85042dc5bfb0b/assets/js/material-components/index.js#L22-L46